packages/astro/src/core/build/static-build.ts

```diff
@@ -158,20 +166,29 @@ export async function staticBuild(opts: StaticBuildOptions) {
 	// condition, so we are doing it ourselves
 	emptyDir(astroConfig.dist, new Set('.git'));
 
+	info(opts.logging, 'build', 'Prebundling for SSR...');
 	// Build your project (SSR application code, assets, client JS, etc.)
 	const ssrResult = (await ssrBuild(opts, internals, pageInput)) as RollupOutput;
 
+	info(opts.logging, 'build', dim(`Completed in ${getTimeStat(timer.buildStart, performance.now())}`));
+
+	timer.clientBuild = performance.now();
 	await clientBuild(opts, internals, jsInput);
 
+	info(opts.logging, null, dim(`Completed in ${getTimeStat(timer.clientBuild, performance.now())}`));
+
+	timer.generate = performance.now();
 	// SSG mode, generate pages.
 	if (staticMode) {
 		// Generate each of the pages.
 		await generatePages(ssrResult, opts, internals, facadeIdToPageDataMap);
 		await cleanSsrOutput(opts);
 	} else {
+		info(opts.logging, null, `\n${bgMagenta(black(' finalizing server assets '))}\n`);
 		await generateManifest(ssrResult, opts, internals);
 		await ssrMoveAssets(opts);
 	}
+	info(opts.logging, null, dim(`Completed in ${getTimeStat(timer.generate, performance.now())} (includes setup + teardown)\n`));
```

Comment:
Gotcha! I included it in case anyone adds up all the `+Xms` logs and notices they _don't_ add up to the total. I agree it's more detail than necessary though.